### PR TITLE
DeleteFileById endpoint

### PR DESCRIPTION
**Pull request by lalagvana**
Added DeleteFileById method and updated packages.

### CODE REVIEW

File: src/FileService.Data/FileRepository.cs [hunk 1]
```diff
@@ -23,13 +24,26 @@ public Guid AddNewFile(DbFile file)
...
+        public void DeleteFileById(Guid fileId)
+        {
+            var dbFile = provider.Files.FirstOrDefault(file => file.Id == fileId);
+
+            if (dbFile == null)
+            {
+                throw new NotFoundException("File with this id was not found.");
+            }
+
+            provider.Files.Remove(dbFile);
```
Comment (DarkFelas) [inline src/FileService.Data/FileRepository.cs:36]: Rework, IsActive = false

**Review by DarkFelas — CHANGES_REQUESTED** [review]

File: src/FileService/FileService.csproj [hunk 2]
```diff
@@ -7,10 +7,10 @@
...
 
   <ItemGroup>
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.2" />
-    <PackageReference Include="MassTransit.AspNetCore" Version="7.0.3" />
-    <PackageReference Include="MassTransit.Extensions.DependencyInjection" Version="7.0.3" />
-    <PackageReference Include="MassTransit.RabbitMQ" Version="7.0.3" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.9" />
+    <PackageReference Include="MassTransit.AspNetCore" Version="7.0.4" />
+    <PackageReference Include="MassTransit.Extensions.DependencyInjection" Version="7.0.4" />
+    <PackageReference Include="MassTransit.RabbitMQ" Version="7.0.4" />
```
Comment (djino97) [inline src/FileService/FileService.csproj:13]: remove

File: test/FileService.Business.UnitTests/DeleteFileByIdCommandTests.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        [Test]
+        public void ShouldThrowExceptionWhenRepositoryThrowsIt()
+        {
+            repositoryMock.Setup(x => x.DisableFileById(It.IsAny<Guid>())).Throws(new Exception());
+
+            Assert.Throws<Exception>(() => command.Execute(fileId));
+        }
+
+        [Test]
+        public void ShouldDeleteFile()
```
Comment (djino97) [inline test/FileService.Business.UnitTests/DeleteFileByIdCommandTests.cs:35]: DisabledFile

File: src/FileService/Controllers/FileController.cs [hunk 4]
```diff
@@ -22,5 +22,13 @@ public File GetFileById([FromServices] IGetFileByIdCommand command, [FromQuery]
         {
             return command.Execute(fileId);
         }
+
+        [HttpDelete("deleteFileById")]
```
Comment (djino97) [inline src/FileService/Controllers/FileController.cs:26]: disableFileById

File: src/FileService.Broker/FileService.Broker.csproj [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+    <TargetFramework>netstandard2.1</TargetFramework>
     <AssemblyName>LT.DigitalOffice.FileService.Broker</AssemblyName>
     <RootNamespace>LT.DigitalOffice.FileService.Broker</RootNamespace>
     <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.2" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.9" />
     <PackageReference Include="MassTransit" Version="7.0.4" />
```
Comment (djino97) [inline src/FileService.Broker/FileService.Broker.csproj:12]: remove

**Review by djino97 — APPROVED** [review]

**Review by DarkFelas — APPROVED** [review]

**Pull request merged**